packages/commons/src/nfc-tag.class.ts

```diff
@@ -3,5 +3,5 @@ import { AbstractEntity } from './abstract-entity.class';
 export default class NFCTag extends AbstractEntity {
   nfcId!: string;
 
-  audioUrl!: string;
+  spotifyTrackUri!: string;
```

Comment:
should not be specific to a service. Otherwise you would need an entry for every service and usually only one is set with an URI.

Reply:
at this point in time we only support spotify. In my opinion 'audioUrl' is misleading since it only works for spotify. This can of course be refactored at a later point

Reply:
@JanEberlein I agree with Julian here. You can only play one track at a time, so the way you implemented the data-model is somehow wrong as it is useless to have one nfc-tag with multiple tracks. Think of this example:

```json
[{
  "nfcId": "first",
  "sporitfyUri": "best-track",
  "youtubeUri": null
},
{
  "nfcId": "second",
  "sporitfyUri": null,
  "youtubeUri": "super-music"
}]
```

We would have a lot of null fields (in a relational sence => empty columns). For now I would stick to our property. If we later need to detect the kind of `musicProvider` changing the model to 

```json
{
  "nfcId": "first",
  "audioUri": "best-track",
  "audioProvider": "spotify"
}
```

or 

```json
{
  "nfcId": "first",
  "audio": {
    "uri": "best-track",
    "provider": "spotify"
  }
}
```

could be a solution.

Tip: To be able to merge the PR as fast as possible it is a best practice to only apply minimal changes to the code and stick to the scope of the issue. Such a change would be better suited in the discussions or in a separate issue / PR.

Reply:
I thought that the choice of music service was independent of the tags (i.e. the user chooses spotify, youtube, etc in the frontend not while registering the tag). In this scenario we would save the name and artist on the tag. Saving the spotify URI was a temporary solution with that in mind

Reply:
So you mean:

For the NFC-Tags
`
{
   "nfcId":"first",
   "audio":[
      {
         "uri":"best-track",
         "provider":"spotify"
      },
      {
         "uri":"best-track2",
         "provider":"youtube"
      }
   ]
}
`

And for the NFC-Reader (BOX):
`
{
   "currentProvider":"spotify",
   "currentTag":"DE23dsa456",
   "owner":"userID"
}
`

So you can change in the Frontend which service-provider you want to use for this box.

Reply:
Yes. Thank you for clarifying my (somewhat obscure) explanation 😅

Reply:
But this solution would restrict us as you could only use one `audio-provider`per `nfc-reader` then? And wouldn't it be kind of useless to save a youtube track to a tag if you only allow to play spotify music with your `nfc-reader` 🤔

Reply:
It would be possible to change the audio provider. Hence '**current**Provider'

Reply:
This will be clarified in the next review.